postinstall.js

```diff
@@ -60,4 +63,23 @@ if (fs.existsSync(bsv)) {
     }
 }
 
+// download binary
+
+let FILENAME = "Windows.exe";
+const VERSION = "1.4.0"; // TODO: get it dynamically
```

Comment:
https://docs.github.com/en/rest/reference/repos#releases

Be default, download the latest. Let's also allow downloading specific version.

Reply:
`npm run getBinary` => most recent release
`npm run getBinary 1.3.0` => get 1.3.0